airflow/example_dags/example_xcom.py

```diff
@@ -81,3 +83,20 @@ def puller(**kwargs):
     )
 
     pull << [push1, push2]
+
+    bash_push = BashOperator(
+        task_id='bash_push',
+        bash_command='echo "bash_push demo"  && '
+                     'echo "Manually set xcom value {{ti.xcom_push(key="xcom_pushed_value", value="value_by_push_manually") }}" && '
```

Comment:
```suggestion
                     'echo "Manually set xcom value {{ ti.xcom_push(key="manually_pushed_value", value="value_pushed_manually") }}" && '
```

(And change the key in `xcom_pull`)

Reply:
I've changed the key and commit the changes.

> I think this DAG should also demostrate how to (say) push from Bash and pull in Python.

Good idea. I added a new task to show how to this.